<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QPushButton" name="buttonHelp">
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>481</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tabCompany">
       <attribute name="title">
        <string>Company Info</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QPushButton" name="buttonRefreshCompany">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QTextBrowser" name="textCompany"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabVehicles">
       <attribute name="title">
        <string>Vehicle Info</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="5">
         <widget class="QTextBrowser" name="textVehicle"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="buttonRefreshVehicle">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineSearchVehicle"/>
        </item>
        <item row="1" column="2">
         <widget class="QRadioButton" name="radioVehicleID">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QRadioButton" name="radioVehicleName">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabLaunchpad">
       <attribute name="title">
        <string>Launchpad Info</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="5">
         <widget class="QTextBrowser" name="textLaunchpad"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="buttonRefreshLaunchpad">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineSearchLaunchpad"/>
        </item>
        <item row="1" column="2">
         <widget class="QRadioButton" name="radioLaunchpadID">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QRadioButton" name="radioLaunchpadName">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabLaunches">
       <attribute name="title">
        <string>Launches</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="2">
         <widget class="QRadioButton" name="radioLaunchesFlightNo">
          <property name="text">
           <string>Flight number</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QRadioButton" name="radioLaunchesLatest">
          <property name="text">
           <string>Latest</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineSearchLaunches"/>
        </item>
        <item row="1" column="3">
         <widget class="QRadioButton" name="radioLaunchesYear">
          <property name="text">
           <string>Launch year</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QRadioButton" name="radioLaunchesUpcoming">
          <property name="text">
           <string>Upcoming</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="buttonRefreshLaunches">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="7">
         <widget class="QTextBrowser" name="textLaunches"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>583</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
